src/Enums/Color.php

```diff
@@ -134,8 +134,6 @@ final class Color
 
     public const RED_900 = '#7f1d1d';
 
-    public const ORANGE = 'orange';
```

Comment:
Shouldn't orange be something here thought? Like, the default orange color? IF not, you can merge @xiCO2k